docs/src/environments/search-engine-visibility.md

```diff
@@ -47,14 +49,20 @@ platform environment:info -e <ENVIRONMENT_NAME> restrict_robots true
 
 ## How it's done
 
-Environments are hidden by including a `X-Robots-Tag` header:
+Search engines are turned away from environments by including a `X-Robots-Tag` header:
 
 ```txt
 X-Robots-Tag: noindex, nofollow
 ```
 
-That tells search engines to not index these sites and not traverse links from these sites.
-This keeps non-production sites out of search engine indexes.
+That tells search engine indexers to not index these sites and not traverse links from these sites.
+This helps keeps non-Production sites out of search engine indexes.
```

Comment:
:lady_beetle:
Typo?
```suggestion
This helps to keep non-Production sites out of search engine indexes.
```

Reply:
Yes, but `to keep` there is British English, so I'll go with just `keep`.